### PR TITLE
Created FAQ.md

### PR DESCRIPTION
**Pull request by neelimagoogly**
I have created FAQs  for fAIr  wrt issue #128 . @kshitijrajsharma , @omranlm and @petya-kangalova please review it.

### CODE REVIEW

File: docs/FAQ.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+## Users
+**Q : What is fAIr?**
+
+A : fAIr is an open-source toolkit developed by the Humanitarian OpenStreetMap Team (HOT) that enables the integration of artificial intelligence (AI) into humanitarian mapping workflows. It provides AI models and tools to automate mapping tasks, improving efficiency and accuracy.
+
+<hr>
+
+**Q : How can I use fAIr in my mapping projects?**
+
+A : To use fAIr, you can follow the installation instructions provided in the documentation. Once installed, you can utilize the pre-trained AI models for tasks such as road network extraction, building footprint detection, or land use classification. The documentation offers examples and guidelines to help you get started.
```
Comment (kshitijrajsharma) [inline docs/FAQ.md]: Add a clause here , Building for now

File: docs/FAQ.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+<hr>
+
+**Q : What types of data can I use with fAIr?**
+
+A : fAIr primarily works with satellite imagery and aerial photographs. You can use various types of imagery sources, including public satellite imagery datasets, to perform mapping tasks using the AI models provided. The documentation provides information on the compatible data formats and recommended data sources.
```
Comment (kshitijrajsharma) [inline docs/FAQ.md]: Various source of imagery which compiles with opensource imagery license

File: docs/FAQ.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+**Q : How can I contribute feedback or report issues with fAIr?**
+
+A : You can contribute to the improvement of fAIr by providing feedback, reporting issues, or suggesting new features through the project's GitHub repository. By creating an issue, you can share your observations or difficulties faced while using fAIr, allowing the developers to address them and enhance the toolkit.
+
+<hr>
+
+**Q : Can I integrate fAIr with other mapping tools or platforms?**
+
+A : Yes, fAIr is designed to be modular and can be integrated into existing mapping workflows and platforms. It provides interoperability with tools like OpenStreetMap, QGIS, and ArcGIS. You can leverage the AI models from fAIr within these tools or use the outputs of fAIr in combination with other mapping datasets and tools.
```
Comment (kshitijrajsharma) [inline docs/FAQ.md]: It provides modular pip packages/ API endpoints that can be used in other scripts and can be integrated as plugins on different platforms such as Qgis, Josm etc

File: docs/FAQ.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+**Q : How can I contribute to the development of fAIr?**
+
+A : You can contribute to fAIr's development by following the guidelines provided in the project's GitHub repository. You can contribute code, suggest new features, or help improve documentation. By submitting pull requests, you can propose changes and collaborate with the project maintainers and the community.
+
+<hr>
+
+**Q : What programming languages and frameworks are used in fAIr?**
+
+A : fAIr frontend is made with React and backend with Django.fAIr is primarily developed using Python, which is widely used in machine learning and AI. It leverages popular frameworks such as PyTorch and TensorFlow for model development and training. Familiarity with Python and these frameworks will be beneficial if you plan to work on fAIr's development.
```
Comment (kshitijrajsharma) [inline docs/FAQ.md]: Rightnow only tensorflow

File: docs/FAQ.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+
+**Q : Are there any specific coding standards or guidelines for contributing to fAIr?**
+
+A : Yes, fAIr follows coding standards and guidelines outlined in the project's repository. These guidelines cover aspects such as code formatting, documentation, and testing. Adhering to these standards helps maintain code consistency and facilitates collaboration among contributors.
+
+<hr>
+
+**Q : Can I propose new AI models or enhancements to existing models?**
+
+A : Absolutely! The fAIr project welcomes contributions related to new AI models or enhancements to existing models. If you have developed a new model that aligns with the project's goals or if you have ideas to improve the current models, you can propose them through the GitHub repository and engage in discussions with the community or you can create them at [here](https://fair-dev.hotosm.org/ai-models) .
```
Comment (kshitijrajsharma) [inline docs/FAQ.md]: Advised to Remove last or part , once approved community will be able go use it , link them to fAIr-utilities

File: docs/FAQ.md (hunk 1 above)
Comment (neelimagoogly) [reply docs/FAQ.md]: Done

File: docs/FAQ.md (hunk 2 above)
Comment (neelimagoogly) [reply docs/FAQ.md]: Done

File: docs/FAQ.md (hunk 3 above)
Comment (neelimagoogly) [reply docs/FAQ.md]: Done

File: docs/FAQ.md (hunk 4 above)
Comment (neelimagoogly) [reply docs/FAQ.md]: Done.

File: docs/FAQ.md (hunk 5 above)
Comment (neelimagoogly) [reply docs/FAQ.md]: Removed the last or part . But I am unable to get the fAIr utilities.

Comment (omranlm) [reply docs/FAQ.md]: @neelimagoogly fAIr repo is the one handing backend and frontend components .. However, the backend is using s separate library we call it fAIr-utilities to handle:

1. Data preparation for the models 
2. models trainings
3. inference process
4. post processing (converting the predicted features to geo data)

Comment (neelimagoogly) [reply docs/FAQ.md]: Thanks @omranlm . I have made the change and linked it to fAIr utilities.

**Comment by neelimagoogly** [comment]
@kshitijrajsharma and @omranlm , I have made the suggested changes and it is ready to merge now . Please review it and merge it if possible.

**Comment by kshitijrajsharma** [comment]
This PR has also changes for Readme , I guess it was based on previous branch ? @neelimagoogly

**Comment by neelimagoogly** [comment]
> This PR has also changes for Readme , I guess it was based on previous branch ? @neelimagoogly

Yes , this PR contains the updated frontend Readme according to PR #134 which you had merged last week . The changes that were made in that PR are reflected here.

**Review by kshitijrajsharma — APPROVED** [review]

**Pull request merged**